rpc/runtime-api/Cargo.toml

```diff
@@ -0,0 +1,42 @@
+[package]
+name = "peaq-pallet-rbac-runtime-api"
+version = "0.0.2"
+edition = "2021"
+description = "A pallet that handles peaq RBAC managment on peaq network."
+authors = ["peaq network <https://github.com/peaqnetwork>"]
+homepage = "https://peaq.network/"
+license = "Apache-2.0"
+publish = false
+repository = "https://github.com/peaqnetwork/peaq-pallet-rbac/"
+
+[dependencies.codec]
+default-features = false
+features = ["derive"]
+package = "parity-scale-codec"
+version = "3.0.0"
+
+[dependencies.sp-api]
+default-features = false
+git = "https://github.com/peaqnetwork/substrate"
+branch = "peaq-polkadot-v0.9.29"
+
+[dependencies.sp-std]
+default-features = false
+git = "https://github.com/peaqnetwork/substrate"
+branch = "peaq-polkadot-v0.9.29"
+
+[dependencies.peaq-pallet-rbac]
+default-features = false
+path = "../../pallet"
+
+[dev-dependencies]
```

Comment:
Do we need this?

Reply:
Including the pallet in the runtime-api? Yes, because of the return type definitions... The runtime-api now returns the same data types as the pallet via extrinsic calls

Reply:
Oh, sorry for the misunderstanding. I mean the dev-dependencies.

Reply:
Ah :+1: One of those copy&paste things... I imported this dependency from the pallet-did example, but did not now, why this is imported there... I will remove it :+1: